lib/machine.js

```diff
@@ -341,6 +341,13 @@ Machine.prototype.$evaluate = function(transformedCode) {
   return this;
 };
 
+Machine.prototype.$evaluate2 = function(transformedCode) {
+  this.context.evaluate(transformedCode);
+  this.$setHalted(false);
+  this.context.evaluate('__runner.init(__top__());');
+  return this;
+};
```

Comment:
This allows to fail fast on a test `Machine#run should handle iterators` otherwise it fails with timeout.  
I will remove this now and undo array.js changes, just will remove array loading.